Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.6
+
+# Dependencies
+RUN go get github.com/gorilla/securecookie && \
+    go get github.com/gorilla/sessions && \
+    go get github.com/hoisie/mustache && \
+    go get github.com/unixpickle/f1le
+
+# Application install
+RUN go install github.com/unixpickle/f1le
```

Comment:
Actually, the `go install f1le` line isn't even necessary. `go get` does that. The README's instructions could be greatly, greatly simplified.